extensions/grpc/deployment/src/main/java/io/quarkus/grpc/deployment/GrpcServerProcessor.java

```diff
@@ -457,6 +458,7 @@ void gatherGrpcInterceptors(BeanArchiveIndexBuildItem indexBuildItem,
     @BuildStep
     @Record(value = ExecutionTime.RUNTIME_INIT)
     @Consume(SyntheticBeansRuntimeInitBuildItem.class)
+    @Consume(BeanInitializedBuildItem.class)
```

Comment:
Hm, how does `@Consume` behave in case of a `MultiBuildItem`? I mean is the behavior equivalent to a `List<BeanInitializedBuildItem>` param?

CC @dmlloyd